x-pack/plugins/security_solution_ess/public/upselling/register_upsellings.tsx

```diff
@@ -58,10 +71,10 @@ export const registerUpsellings = (
     {}
   );
 
-  const upsellingSectionsToRegister = upsellingSections.reduce<SectionUpsellings>(
+  const upsellingSectionsToRegister = upsellingSections(services).reduce<SectionUpsellings>(
```

Comment:
This should not be necessary if the component is wrapped with the service provider. You can compose the package component with a small wrapper component, so you can access services and call `getUrlForApp`.